submarine-cloud-v2/controller.go

```diff
@@ -1134,6 +1136,17 @@ func (c *Controller) syncHandler(workqueueItem WorkQueueItem) error {
 		if err != nil {
 			return err
 		}
+
+		// Port-forwarding
+		// TODO:
+		//   (1) multi-tenant port-forwarding
+		//   (2) Basic operations: on/off/modify (change port)
+		//   (3) in-cluster
+		if action == ADD {
+			if !incluster {
```

Comment:
Here the `incluster` still works. But the `incluster` here is actually the global variable in `main.go`. I think what you want to do is to add this `incluster` as one attribute of `struct Controller` and use it.

Reply:
I have fixed it.